### DISCUSSION TITLE
Cameras turn black when they are scrolled into view

### DISCUSSION THREAD

**Discussion by dalcon007** [blakeblackshear/frigate#12045] [Report a Bug] [answered]
### Describe the problem you are having

Hello,

Running Frigate on Home Assistant. I have 6 cameras. They appear black on the UI (with a red recording symbol). They sometimes appear just as they're about to go off screen when scrolling. Sometimes all of my camera feeds appear as expected, usually after a restart but not always. Tapping on the camera will produce a black screen, and sometimes the video feed will appear.

### Steps to reproduce

On the main screen, scrolling the view will sometimes show the cameras as they are off screen. I suspect this could be a configuration issue with go2rtc?

### Version

0.14.0-4e5a6eb

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

Amcrest AD410, Hikvison DS-2CD2347G1-LU, Reolink 

### Any other information that may be helpful

I've set encoding to h264 on all cameras that support it

**Comment by NickM-27** [comment, marked as answer]
this has already been fixed in later commits

**Reply by dalcon007** [reply]
Okay, great, thank you. Do I just need to wait for the next beta release or can I somehow update my instance now?

**Reply by NickM-27** [reply]
you can use the dev image https://github.com/blakeblackshear/frigate/pkgs/container/frigate
